sbc_translation/models/translate_connector.py

```diff
@@ -29,13 +29,13 @@ def __init__(self):
 
         self.current_time = datetime.datetime.now()
 
-    def upsert_text(self, sponsorship, letter):
+    def upsert_text(self, letter, letter_name):
         """Push or update text (db table) on local translate platform
         """
-        child = sponsorship.child_id
-        sponsor = sponsorship.partner_id
+        child = letter.sponsorship_id.child_id

```

Comment:
I think child_id and partner_id are directly present in letter object. So you can do `letter.child_id` :wink: